### ISSUE TITLE
[Feature Request] Buff special stack count x/y offset.

### ISSUE THREAD

**Issue by TheDescender** [Tercioo/Plater-Nameplates#235]
Seems to not be an option present to move the stack counter on special marked buffs/debuffs (haven't checked for normal buffs). 

Would be nice to have this feature, as with the current location of my special buff icons, the health bar blocks half the stack count number.

**Comment by TheDescender** [comment]
I feel stupid for asking this, since it's just a function call with some variables, but am I understanding correctly, that this is how this should be implemented?

![image](https://github.com/user-attachments/assets/84ad095b-2060-441d-8bd9-7d60e7d7ec0c)

As you can see from the image, that's not working.

**Comment by TheDescender** [comment]
I'll just reboot. On a side note, if you're ever looking to implement offsets as an option, I notice the stack count strata is also lower than the icon swipe strata, so the dark filter will cover the counter.

**Comment by TheDescender** [comment]
Nope, neither reload or restart seems to make the stack count move. This is working on your end?

**Comment by TheDescender** [comment]
Ok I managed to get this working from the Nameplate Updated Hook instead of the constructor. And reverse engineer some extra settings from going through the WeakAura's docs.

```
function (self, unitId, unitFrame, envTable)
    unitFrame.ExtraIconFrame:SetOption ("stack_text_anchor", "top")
    unitFrame.ExtraIconFrame:SetOption ("stack_text_rel_anchor", "bottom")
    unitFrame.ExtraIconFrame:SetOption ("stack_text_x_offset", 0)
    unitFrame.ExtraIconFrame:SetOption ("stack_text_y_offset", 30)
    unitFrame.ExtraIconFrame:SetOption ("cooldown_swipe_enabled", false)
    --unitFrame.ExtraIconFrame:SetOption ("show_text", false)
    --unitFrame.ExtraIconFrame:SetOption ("cooldown_edge_texture", "Interface\Cooldown\transparent")
    unitFrame.ExtraIconFrame:SetOption ("text_y_offset", 0)
end
```

For anyone running into this, with these settings the counter is centered to the top of the icon. show_text disables the CD text if u want that to happen. As you can guess, cooldown_edge_texture is a string that refers to an image path with the cooldown edge texture. You cannot disable the Edge texture, unless you add a custom transparent image. Otherwise it'll throw a white picture over your icon. I didn't bother doing this, but it'll probably work. You can guess what the rest does.

**Comment by cont1nuity** [comment]
And how would WA docs help here?

**Comment by TheDescender** [comment]
Yeah Added seems to also work fine, Thanks! As far as I can tell, I have nothing else that interacts with my buff special window, especially not the stack counter. I tested it in constructor on a fresh plater install, which had the same issue. No clue why that doesn't work. I'll edit my post to say I put it under Added.

WA docs just helped me find show_text, cooldown_edge_texture, cooldown_swipe_enabled, etc. If these are not standard, count me lucky I guess.

**Comment by cont1nuity** [comment]
There is no option for that. I will need to have a look at that internally for a potential fix.
